### DISCUSSION TITLE
[Bug] Unable to login “Google couldn't confirm this attempt to sign in is safe. If you think this is a mistake, you can close and try again to sign in.”

### DISCUSSION THREAD

**Discussion by qnblackcat** [qnblackcat/uYouPlus#428] [General]
Refer to #420

**Comment by CycilismRSA** [comment]
Has this been resolved? Im having the same issue iphone xr ios 15.1

**Reply by baobaoit** [reply]
This issue isn't resolved yet.

**Comment by the0neyouseek** [comment]
Using a (paid) wildcard cert to use the bundle id to `com.google.ios.youtube` doesn't seem to fix this either

**Comment by arichornlover** [comment]
I wish this was fixed I really need to sign in without needing to copy the link and putting it on the browser. Google is strict with these changes. and I think google really trying to stop people like us from using uYou/Cercube and uYouPlus/CercubePlus.
Edit: I realized the copy and paste to url trick no longer works.

**Comment by PoomSmart** [comment]
It looks like Google finally will reject sign in request without `device_challenge_request` on some accounts.

**Comment by thpryrchn** [comment]
Workaround that worked for me: 

I installed uYouPlus_17.14.2_2.1.ipa, Then signed into my google account, and it works on that old version. 

Then I install the latest version **over** the old version, and it retains the login info.

Works with AltStore and TrollStore!

Edit: clarification and formatting.

**Reply by baobaoit** [reply]
I can confirm this workaround is worked for me either.

More information:
Device: iPhone 11 Pro Max
iOS: 15.1

Reproduce:
Step 1: install uYouPlus_17.14.2_2.1.ipa
Step 2: sign in to your google account like normal
Step 3: Now, you can install the latest version of uYouPlus

Thanks @thpryrchn

**Reply by arichornlover** [reply]
So this version might be a good workaround, but the only downside is that it probably works with 17.14.2 or older. Since I’m not a jailbreaking user I will be completely wrong about it.

**Reply by baobaoit** [reply]
Hi @arichorn,

This is not related to jailbreak or not. It is compatible with sideloading (e.g. AltStore) either.

**Reply by baobaoit** [reply]
Hi @qnblackcat,

Can you please double-check and update this workaround or some update for uYouPlus?

Thank you.

**Reply by minhhieu184** [reply]
@baobaoit Is it necessary to uninstall 17.14.2_2.1 before install the latest version ?
Thank you.

**Reply by baobaoit** [reply]
Hi @minhhieu184,

You don't need to uninstall the uYouPlus_17.14, just install the latest.

**Reply by the0neyouseek** [reply]
Doesn't work on iOS 16 (latest beta) sadly

**Reply by arichornlover** [reply]
@baobaoit I meant like, jailbreaking to be able to see the methods of what YouTube they use and such

**Reply by hektor2107** [reply]
Doesn't work on iOS 15.6.1 iPhone 13 pro

**Reply by ZomkaDEV** [reply]
Doesn't work on iOS 15.2.1, iPhone 13 mini

**Reply by djmuted** [reply]
Confirmed working on iOS 15.1 with TrollStore

**Reply by thpryrchn** [reply]
> Doesn't work on iOS 15.2.1, iPhone 13 mini

Did you use AltStore? Do not uninstall the older version.  Update by installing new version ove old version.

**Reply by thpryrchn** [reply]
> Confirmed working on iOS 15.1 with TrollStore

That is what I used :)

**Reply by lukman4** [reply]
Can confirm this works for me.

Device information:
iPhone 13 Pro Max
iOS: 15.0
Install method: TrollStore

**Reply by ZomkaDEV** [reply]
> > Doesn't work on iOS 15.2.1, iPhone 13 mini
> 
> Did you use AltStore? Do not uninstall the older version. Update by installing new version ove old version.

I use AltServer_Linux. I uninstalled the latest version, installed 17.14.2_2.1, tried logging in and got the same error :(

**Reply by ZomkaDEV** [reply]
I will try it again tomorrow though.

**Reply by thpryrchn** [reply]
> Doesn't work on iOS 16 (latest beta) sadly

Are you using AltStore? Make sure you do not uninstall the old version. Just install the newest version on top of the old one I used in the workaround. 

Ps, other older versions may work. That is just the one I still had in my downloads on my iCloud.

**Reply by Micha3lo** [reply]
Still throws the same error as before on the older version mentioned in the comments

iPhone 13 Pro
iOS 15.6.1
AltStore installation

**Reply by the0neyouseek** [reply]
> Ps, other older versions may work. That is just the one I still had in my downloads on my iCloud.

Sadly I tried the oldest iOS 16 working version and I still get the error message trying to login. I'll wait for the fix

**Reply by ZomkaDEV** [reply]
@thpryrchn did you have uYou+ installed before on your device? Mine 13 mini didn't have installed uYou+ until today when I kept getting this error, do you think that could be related to the issue?

**Reply by DeltaOn3X** [reply]
**It actually worked for me. I used Trollstore**. 

_(I have a working Cercube version installed with Alstore tho)_

I've installed uYou+ 17.14.2, signed in and installed the latest version over (DID NOT DELETE THE OLD VERSION).

After installing the latest version over 17.14 im still signed in. Thank you very much!!! Im gonna try it with Cercube+ and tell you which old version worked for me.

**Reply by DeltaOn3X** [reply]
So i tried it with Cercube+

I deleted uYou+ in Trollstore, installed CercubePlus_17.13.3_5.3.9.ipa, i was auto signed in. I installed the latest version over it in trollstore and was still signed in. 

I dont know what happens when i sign out of the old version (17.13.3). 

You can try it out and give us feedback if you want :)

**EDIT:**

I dont know whats going on but i have deleted every youtube app i had and just installed the latest version of cercube. I suddenly was auto signed in. I deleted Cercube+ and installed uYou+, auto signed in. Dont know whats going on now :D

**Reply by showmak** [reply]
I followed your workaround and it worked. Thanks

**Reply by maybehieu** [reply]
Can confirm this work for me, details below
Reproduce with:
- Latest TrollStore ipa
- (I reboot the phone here, might be optional)
- Install uYou17.15.2_2.1 and login into Google account
- Install lastest uYou version (17.35.3_2.1) and it works!
iPhone X (iOS 15.0), only instaling the lastest patch/ version of uYou will still result with the error (if someone needs the info)
Cheers

**Reply by truongtv22** [reply]
Thanks @maybehieu

**Reply by TheOldMonk90** [reply]
Working on iPhone XR 15.0 with TrollStore. Thank you very much 😇

**Reply by jimmysofat6864** [reply]
Was not able to get this to work for me. Using iPhone 13 on 15.6.1. Downgrading app version and attempting sign in didn't work. Not even pasting the link in a private browsing tab worked.

**Reply by ComeOnBear** [reply]
Working on iPhone SE 2020 (A13) iOS 15.0! Thanks!

**Comment by hobgoblin123** [comment]
Am facing this issue as well any workaround would be much appreciated

**Reply by arichornlover** [reply]
This is honesty a impressive  workaround, the extra steps are fine though.
Just hope YouTube doesn’t push out a server side patch

**Reply by baobaoit** [reply]
Hi @hobgoblin123,

This workaround worked for me, you can reference: https://github.com/qnblackcat/uYouPlus/discussions/428#discussioncomment-3545549

I hope Google will not patch it on the server side.

**Reply by hobgoblin123** [reply]
Thank You

**Comment by CycilismRSA** [comment]
Thanka for all these, ill try them, i have a older version of uyouoplus that i will try and sync data from

**Comment by ZomkaDEV** [comment]
This is very weird to say the least... I have tried using an older version of uYouPlus, copy and pasting the URL link and using CercubePlus - it did not help, sadly. I'm using an iPhone 13 mini on iOS 15.2.1 :(

**Comment by arichornlover** [comment]
This sign in issue still happens whenever sideloading a YouTube ipa without any tweaks in it. Google went too far with this change

**Comment by dnscherry** [comment]
I just changed my phone and this is happening , tried all free methods but nothing worked. Hope it gets a fix soon

**Reply by x90skysn3k** [reply]
same, workarounds did not work for me, i think because i am on a new phone

**Reply by maybehieu** [reply]
@x90skysn3k you can try [#428](https://github.com/qnblackcat/uYouPlus/discussions/428#discussioncomment-3545549), it works for me
edit: change hyperlink text

**Comment by ifRemix** [comment]
It helped to install Cercube and uYou put on top of it.👍🏼

**Reply by NeilRiver** [reply]
no isnt

**Reply by ifRemix** [reply]
But it was enough for one day. Today the authorization was lost again.

**Comment by vetrrrix** [comment]
nothing worked, tried everything written in here

**Reply by ifRemix** [reply]
It's sad... it worked the first time for me.  ¯\_(ツ)_/¯

**Reply by showmak** [reply]
This workaround worked for me https://github.com/qnblackcat/uYouPlus/discussions/428#discussioncomment-3545549

**Reply by vetrrrix** [reply]
I have this version and still it is not possible to login :/

**Comment by gh0sti** [comment]
Does signing with signulous resolve this? Or another signing server to upload the ipa to?

**Comment by NoMorePremium** [comment]
Hope I'm not adding to the echo chamber....

I've tried the method of using an older version to sign in, but alas it doesn't work. The only solution I have, which isn't much of a solution at all is to search your username and use the playlists you've created. This really only applies to people who use the app for musical purposes.

This change is something awful. Something really awful.

**Comment by vantud** [comment]
Problem still present in latest version and not success with workarounds. (Iphone SE, iOS 15.6.1)

**Comment by Milchkarl** [comment]
I installed this version with trollstore on iPhone 9,3 (iPhone 7 GSM) on iOS 14.5.1 and for me it also does not work even though the release notes said that using trollstore will bypass the error.

**Comment by axemre** [comment]
Currently, there seems to be no workaround, the only way to get "logged in" functionality is to create a new account, that's what I've done. Google seems to block these third-party YouTube apps, the link which we are copying to log in (which we use to log in via Safari) shows that there's some "blockedsignin" or "embeddedsignin" "feature?". The only way (currently) to "fix" that is to create a new account in the window opened by YouTube. (Sign-in windows)
If you sign out of this freshly created account, it'll never allow to log in again, even with the new account. Google went too far with that change..

**Comment by fraces** [comment]
Same here, Tried everything though AltStore, even downgrading, but nothing works. It's utterly annoying seeing my library messed up, i think this might be a deal-braker.

The only way I managet
To login, was to create a new account and subscribe again to all the content I am I interested in, but seems a temporary solution (and a frankly annoying one) 

Iphone XS
Version 15.6.1

**Comment by qnblackcat** [comment]
This workaround should do the job: #447

**Reply by CountDer3k** [reply]
Can confirm this works:
iPhone 14 Pro Max (iOS 16)

**Reply by FrogDadd** [reply]
Doesnt work as of today. The shortcut causes a permanent loop of "Do you want to log in with google" until you stop it, crashing both youtube apps.

**Comment by AuroraWright** [comment]
Btw if anyone who's experienced with iOS RE wants to check, I'm pretty sure that if the checks fail when you install an IPA with the same package name as the app store they're probably checking if the app is encrypted https://stackoverflow.com/a/12976087
